### PR TITLE
Update chart to be Kubernetes 1.19+ compatible

### PR DESCRIPTION
**Pull request by DavisRayM**
# Changes

- Update ingress template to be compatible with Kubernetes API v1.19+

### CODE REVIEW

File: templates/ingress.yaml [hunk 1]
```diff
@@ -1,8 +1,10 @@
 {{- if .Values.ingress.enabled -}}
 {{- $fullName := include "enketo.fullname" . -}}
 {{- $svcPort := .Values.service.port -}}
-{{- if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
+{{- if semverCompare ">=1.14-0,<1.19" .Capabilities.KubeVersion.GitVersion -}}
```
Comment (evansmurithi) [inline templates/ingress.yaml:4]:
```suggestion
{{- if semverCompare ">=1.14-0,<1.19-0" .Capabilities.KubeVersion.GitVersion -}}
```

File: templates/ingress.yaml [hunk 2]
```diff
@@ -32,10 +34,20 @@ spec:
       http:
         paths:
           {{- range .paths }}
+          {{- if semverCompare ">=1.19" $.Capabilities.KubeVersion.GitVersion }}
```
Comment (evansmurithi) [inline templates/ingress.yaml:37]:
```suggestion
          {{- if semverCompare ">=1.19-0" $.Capabilities.KubeVersion.GitVersion }}
```

**Review by evansmurithi — CHANGES_REQUESTED** [review]

File: templates/ingress.yaml [hunk 3]
```diff
@@ -1,8 +1,10 @@
 {{- if .Values.ingress.enabled -}}
 {{- $fullName := include "enketo.fullname" . -}}
 {{- $svcPort := .Values.service.port -}}
-{{- if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
+{{- if semverCompare ">=1.14-0,<1.19-0" .Capabilities.KubeVersion.GitVersion -}}
 apiVersion: networking.k8s.io/v1beta1
+{{- else if semverCompare ">=1.19" .Capabilities.KubeVersion.GitVersion -}}
```
Comment (DavisRayM) [inline templates/ingress.yaml:6]:
```suggestion
{{- else if semverCompare ">=1.19-0" .Capabilities.KubeVersion.GitVersion -}}
```

**Comment by evansmurithi** [comment]
@DavisRayM can you also add `ingressClassName` in spec e.g.

```yaml
  {{- if and .Values.ingress.className (semverCompare ">=1.18-0" .Capabilities.KubeVersion.GitVersion) }}
  ingressClassName: {{ .Values.ingress.className }}
  {{- end }}
```

**Pull request merged**